HELPS/ClientApp/src/components/user/User.tsx

```diff
@@ -1,16 +1,28 @@
-import {Dispatch} from 'redux';
 import {connect} from 'react-redux';
 import * as React from 'react';
 import {Component} from 'react';
-import {UserDispatchProps, UserProps, UserStateProps} from '../../types/components/UserTypes';
+import {
+    UserDispatchProps,
+    UserProps,
+    UserState,
+    UserStateProps
+} from '../../types/components/UserTypes';
 import {AppState} from '../../types/store/StoreTypes';
 import {retrieveUser, updateUser} from '../../store/actions/UserActions';
 import UserDetailsForm from './UserDetailsForm';
 import Button from 'react-bootstrap/Button';
 import {submit} from 'redux-form';
-import { ThunkDispatch } from 'redux-thunk';
+import {ThunkDispatch} from 'redux-thunk';
 
-class User extends Component<UserProps> {
+class User extends Component<UserProps, UserState> {
+
+    constructor(props: UserProps) {
+        super(props);
+        this.state = {
+            searchTerm: '',
+            loadAllUsers: []
+        };
+    }
 
     componentDidMount() {
         if (this.props.authenticated) {
```

Comment:
You need to call loadAllUsers here, this will set the prop value 'student' to an array of students